report/2.Analysis/Scope.tex

```diff
@@ -0,0 +1,43 @@
+%This section is meant to specify the scope of project, both in terms of the report, the language that will be developed and the compiler.
+\section{Scope}
+As stated in the above analysis, the purpose of this project is to design, create and implement a programming language for the Arduino platform that is simple for beginners to use, whilst still teaching the fundamentals of programming.
+In other terms, the language is meant to enable beginners to quickly and easily write
+code that can be executed, without losing the functionality of a high-level language.
+
+It is imperative to consider the needed functionality in order responsibly compose a language for beginners.
```

Comment:
"imperative" can be changed to "essential" (I don't believe imperative is the right word to use).